spec/unit/show-hide-content.spec.js

```diff
@@ -225,6 +273,10 @@ describe('show-hide-content', function () {
         }))
         expect(events && events.click).toContain(jasmine.objectContaining({
           namespace: 'ShowHideContent',
+          selector: 'input[type="radio"][name="test.single"]'
+        }))
+        expect(events && events.click).toContain(jasmine.objectContaining({
```

Comment:
Could you split this test out?.. You seem to making and boolean evaluation inside an assertion?
Perhaps having a test which check that the event doesn't exist?